scripts/python/sitecustomize.py

```
@@ -0,0 +1,182 @@
+    if not isinstance(raw_preflight, dict):
+        return ()
+    raw_excludes = raw_preflight.get("test_exclude", ())
+    return _normalize_exclude_list(raw_excludes)
+
+
+def _patch_configuration_validation() -> None:
+    """Allow ``[preflight]`` in ``lading.toml`` without raising errors."""
+
+    try:
+        from lading import config as config_module
+    except Exception:  # pragma: no cover - lading unavailable
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)